src/main/java/pl/javastart/zadanie20/UserController.java

```diff
@@ -0,0 +1,53 @@
+package pl.javastart.zadanie20;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@Controller
+public class UserController {
+
+    private final UserRepository userRepository;
+
+    @Autowired
+    public UserController(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @RequestMapping("/users")
+    @ResponseBody
+    public String getAddedUsers() {
+        StringBuilder result = new StringBuilder();
+        List<User> users = userRepository.getUsers();
+        for (User user : users) {
+            result.append(user.toString()).append("<br/>");
+        }
+        return result.toString();
+    }
+
+    @RequestMapping("/add")
+    public String addUserForm(@RequestParam(value = "imie", required = true) String firstName,
+                              @RequestParam(value = "nazwisko", required = true) String lastName,
+                              @RequestParam(value = "wiek", required = true) String ageStr) {
```

Comment:
`required = true` jest wartością domyślną, raczej się tego nie dodaje

Reply:
słuszna uwaga, nawet InelliJ to sugerował, zapomniałem usunąć.